ci/images/setup.sh

```diff
@@ -26,6 +26,15 @@ curl -L ${JDK_URL} | tar zx --strip-components=1
 test -f /opt/openjdk/bin/java
 test -f /opt/openjdk/bin/javac
 
+if [[ $1 != $2 ]]; then
+	SECONDARY_JDK_URL=$( ./get-jdk-url.sh $2 )
```

Comment:
I think I'd prefer toolchain to secondary here. IMO, it ties it together a bit better with the Gradle terminology.

Reply:
Good suggestion - at least when I can get rid of the `$1 != $2`. It felt natural to call this secondary in this case, but I like your suggestion more.